terracumber/terraformer.py

```diff
@@ -26,15 +26,21 @@ class Terraformer:
     terraform_bin - path to terraform bin
     """
 
-    def __init__(self, terraform_path, maintf, backend, variables={}, output_file=False, terraform_bin='/usr/bin/terraform'):
+    def __init__(self, terraform_path, maintf, backend, variables={}, output_file=False, terraform_bin='/usr/bin/terraform', variables_file="", tfvars_files=[]):
```

Comment:
That I guess it could be a matter of taste, but if it was me, I would prefer to only have "variables" as the parameter that handle variables.
Therefore, I would process the content that we are obtaining from variable_files or tfvar_files outside of this class init, if you prefer as a static method in this class, if not outside the class.
And once you process this content, we combine all inside "variables" parameter.